vm/rust/src/lib.rs

```diff
@@ -240,55 +276,78 @@ pub extern "C" fn cairoVMExecute(
 
         let mut txn_state = CachedState::create_transactional(&mut state);
         let fee_type;
+        let minimal_l1_gas_amount_vector: Option<GasVector>;
```

Comment:
could you please refer me to the discussion regarding this or explain what this minimal l1 gas is all about?

Reply:
send you in slack